R/read_field_obs.R

```diff
@@ -1,64 +1,65 @@
-#' @title read in field data into a data.frame

```

Comment:
This is just removing the title tag per my response above. Not sure why the diff doesn't render the new line without the tag but its there in the source.